packages/angular/src/lib/stencil-generated/components.ts

```diff
@@ -564,7 +590,11 @@ export declare interface GcdsInput extends Components.GcdsInput {
    */
   gcdsBlur: EventEmitter<CustomEvent<void>>;
   /**
-   * Update value based on user input.
+   * Emitted when the input has received input.
```

Comment:
Same comment as above
```suggestion
   * Emitted when the element has received input.
```